flink-state-backends/flink-statebackend-forst/src/main/java/org/apache/flink/state/forst/ForStDBGetRequest.java

```diff
@@ -34,12 +34,35 @@ public class ForStDBGetRequest<K, V> {
 
     private final K key;
     private final ForStInnerTable<K, V> table;
-    private final InternalStateFuture<V> future;
+    private final InternalStateFuture future;
 
-    private ForStDBGetRequest(K key, ForStInnerTable<K, V> table, InternalStateFuture<V> future) {
+    private final boolean toBoolean;
+    private final boolean checkMapEmpty;
+
+    private int keyGroupPrefixBytes = 1;
+
+    private ForStDBGetRequest(
+            K key,
+            ForStInnerTable<K, V> table,
+            InternalStateFuture future,
+            boolean toBoolean,
+            boolean checkMapEmpty) {
         this.key = key;
         this.table = table;
         this.future = future;
+        this.toBoolean = toBoolean;
```

Comment:
`toBoolean` seems not very clear. I would suggest to add `ForStDBExistRequest` for this kind of operation.

Reply:
👍，I added `ForStDBMapCheckRequest` for this kind of operation.